### PR TITLE
Rename to dune in README.md

### PR DESCRIPTION
**Pull request by emillon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,28 +4,28 @@ Dune (Jbuilder) - A composable build system
 __Jbuilder has been renamed to Dune__. A full renaming of the documentation and
```
Comment (diml) [inline README.md]: we should probably adapt this sentence

Comment (emillon) [reply README.md]: Yes, and maybe make the "dune (jbuilder)" part less prominent as well.

Comment (diml) [reply README.md]: agreed, we can remove jbuilder from the title

**Comment by emillon** [comment]
Thanks for the feedback!

**Pull request merged**